### ISSUE TITLE
Unable to open nested pouchs as client

### ISSUE THREAD

**Issue by rdelpeso** [cr4shmaster/gitMagicalPouchDST#7]
When enabling pouch-ception and attempting to open a pouch stored inside another pouch instead of opening the nested pouch the current pouch gets closed.

This is not an issue as a host, only as a client.

**Comment by cr4shmaster** [comment]
I hadn't even thought about trying to open a pouch inside another. :)

**Comment by rdelpeso** [comment]
It is a strange thing that it works for the host and not for the client, have u encountered this before? (things working for host and not client)

**Comment by cr4shmaster** [comment]
Definitely. I don't remember what exactly though; it had something to do with crafting. 
I think that Klei was looking into this issue at some point.

**Comment by rdelpeso** [comment]
To further complicate, while testing with a dedicated server found a secondary bug: It is possible to store a pouch inside another pouch ONLY through right clicking with one pouch being hold in the mouse. If you open a pouch then attempt to move a pouch to any of the slots, the action is prevented.

It is good that I am seeing these issues while being a client. This will allow me to do more robust development. It would suck if i was not able to reproduce these issues.

**Comment by cr4shmaster** [comment]
So this too is only happening for the clients?

**Comment by rdelpeso** [comment]
Yup, if i host the game from DST I can open/close fine, nest fine, open and close nested fine. If I join as a client on my Dedicated Server (even as admin) I can open/close fine. I am unable to nest via clicking (only via right click store on the pouch) and the nested pouch will not open it will only close the outer pouch and do nothing else. Let me know when u have a couple minutes and I can demonstrate.

There are some other interesting things. As a client, the function "params.magicpouch.itemtestfn" only gets called when right clicking on the pouch then selecting store. Trying to drop an item on the pouch triggers it too, but not when trying to drop a pouch within a pouch. Is odd.

I wonder if it has something to do with how you originally wired the pouch that is not too client friendly for network communication.

Now that i think about it, clients never had the bug of pouch-ception xD only hosts ever did.

**Comment by cr4shmaster** [comment]
Does only "params.magicpouch.itemtestfn" not get called? Does "params.icepouch.itemtestfn" work as intended?  

Also, what did you mean by this?
> clients never had the bug of pouch-ception xD only hosts ever did

**Comment by rdelpeso** [comment]
I will momentarily disable the ability to enable pouch-ception in the options of the mod as a bug-preventing meassure until I can find a solution for clients. See #10 for details
